android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -118,11 +118,9 @@ class NeuroidReactnativeSdkModule(reactContext: ReactApplicationContext) :
 
     @ReactMethod
     fun attemptedLogin(id: String, promise: Promise) {
-        // TODO
-        // Attempted Login is temporarily disabled until we have a new release for react native
-        // var result = NeuroID.getInstance()?.attemptedLogin(id)
-        // result?.let { promise.resolve(it) }
-        // promise.resolve(false)
+        var result = NeuroID.getInstance()?.attemptedLogin(id)
```

Comment:
Not related to this specific PR, but I think the id param should be an optional string because RN might not send anything

Reply:
updated

Reply:
Also not related to this PR but can you change the same thing in the iOS function to have the string be optional?

Reply:
updated